src/bin/psql/describe.c

```diff
@@ -4625,12 +4622,8 @@ listTables(const char *tabtypes, const char *pattern, bool verbose, bool showSys
 	PQExpBufferData buf;
 	PGresult   *res;
 	printQueryOpt myopt = pset.popt;
-<<<<<<< HEAD
-	static const bool translate_columns[] = {false, false, true, false, false /* Storage */, false, false, false, false};
-=======
 	int			cols_so_far;
-	bool		translate_columns[] = {false, false, true, false, false, false, false, false};
->>>>>>> sync-pg-phase1
+	bool		translate_columns[] = {false, false, true, false, false /* Storage */, false, false, false, false};
```

Comment:
Do we have enough elements in the array? Commit 9a2ea61 added one element.

Reply:
added